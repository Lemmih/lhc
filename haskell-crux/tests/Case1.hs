{-# LANGUAGE UnboxedTuples, MagicHash #-}
module Case1 where

x = case (# 10 , 20 #) of
      (# a, b #) -> a
