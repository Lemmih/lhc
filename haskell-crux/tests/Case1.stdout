Case1.x : LHC.Prim.Int
Case1.x =
  let Case1.a^1:LHC.Prim.Int = LHC.Prim.I#
                                 (LHC.Prim.i64toi32 10)
  let Case1.b^2:LHC.Prim.Int = LHC.Prim.I#
                                 (LHC.Prim.i64toi32 20)
  Case1.a^1
